### PR TITLE
update post-release developer docs

### PR DESCRIPTION
**Pull request by creviera**
## Status

Ready for review

## Description of Changes

Clarify post-release steps in the docs

### CODE REVIEW

File: docs/development/release_management.rst [hunk 1]
```diff
@@ -222,10 +222,22 @@ Release Process
...
-monitor) and SecureDrop community support forum for any issues that users are
-having.
-
-Finally, in a PR back to develop, cherry-pick the release commits (thus ensuring a consistent
-changelog in the future) and bump the version numbers
-in preparation for the next release (this is required for the upgrade testing scenario).
+Now it's time to backport the changelog from the release branch and bump the SecureDrop version so
+that it's ready for the next round of QA testing.
+
+We backport the changelog by cherry-picking any commits that modified `changelog.md` during the
```
Comment (eloquence) [inline docs/development/release_management.rst:228]: These docs are reStructuredText, so you'll need to use 
````rst
``this syntax``
````
for inline code fromatting and
````rst
`this syntax <https://example.com/>`_.
````
to make external links. (There are other ways to make links; see the docs themselves for examples.)

File: docs/development/release_management.rst [hunk 2]
```diff
@@ -222,10 +222,22 @@ Release Process
...
+release. You can look at the file history by checking out the release branch and running: 
+`git log --pretty=oneline changelog.md`. The output will contain the commit hashes associated with 
+the release. Create a new branch based on `develop` and cherry-pick these commits using the
+`-x` flag.
+
+Now you're ready to bump the SecureDrop version on your new branch. There are a bunch of version
+files that'll need to be updated in order to set up the upgrade test for the next release. We do
+this by running the version-updater script and specifying the new version number, which will be the
+next minor version with `~rc1` appended. For example, if the relase is 1.3.0, then you'll run: 
+`./update_version.sh 1.4.0~rc1`.
```
Comment (eloquence) [inline docs/development/release_management.rst:238]: Even though it's almost implicit, I'd still spell out that the next step is to make a PR into `develop`.

File: docs/development/release_management.rst [hunk 3]
```diff
@@ -222,10 +222,22 @@ Release Process
...
+`-x` flag.
+
+Now you're ready to bump the SecureDrop version on your new branch. There are a bunch of version
+files that'll need to be updated in order to set up the upgrade test for the next release. We do
+this by running the version-updater script and specifying the new version number, which will be the
+next minor version with `~rc1` appended. For example, if the relase is 1.3.0, then you'll run: 
+`./update_version.sh 1.4.0~rc1`. 
+
+Now the only thing left to do is to monitor the [FPF support portal](support.freedom.press)
+(also available as a Tor Onion Service at http://support6kv2242qx.onion/) and the
```
Comment (eloquence) [inline docs/development/release_management.rst:241]: I'd skip the .onion address here, it's liable to change and advertised on the support portal itself.

File: docs/development/release_management.rst [hunk 4]
```diff
@@ -222,10 +222,22 @@ Release Process
...
 ------------
 
-After the release, carefully monitor the FPF support portal (or ask those that have access to
-monitor) and SecureDrop community support forum for any issues that users are
-having.
-
-Finally, in a PR back to develop, cherry-pick the release commits (thus ensuring a consistent
-changelog in the future) and bump the version numbers
-in preparation for the next release (this is required for the upgrade testing scenario).
+Now it's time to backport the changelog from the release branch and bump the SecureDrop version so
```
Comment (eloquence) [inline docs/development/release_management.rst:225]: For the avoidance of ambiguity, I would say "bump the SecureDrop version in `develop`" here.

**Comment by eloquence** [comment]
Thanks much for adding this additional detail, clearly the docs were relying on institutional knowledge here. Added a couple of suggested tweaks from my end.

File: docs/development/release_management.rst [hunk 5]
```diff
@@ -222,10 +222,22 @@ Release Process
...
+release. You can look at the file history by checking out the release branch and running: 
+``git log --pretty=oneline changelog.md``. The output will contain the commit hashes associated with 
+the release. Create a new branch based on ``develop`` and cherry-pick these commits using the
+``-x`` flag.
+
+Now you're ready to bump the SecureDrop version on your new branch. There are a bunch of version
+files that'll need to be updated in order to set up the upgrade test for the next release. We do
+this by running the version-updater script and specifying the new version number, which will be the
+next minor version with ``~rc1`` appended. For example, if the release is 1.3.0, then you'll run: 
+``./update_version.sh 1.4.0~rc1``. Accept all the default changes from the script. You'll only need
```
Comment (zenmonkeykstop) [inline docs/development/release_management.rst:238]: This should be run in the dev container: ``securedrop/bin/dev-shell ../update_version.sh 1.4.0~rc1``. Also might be worth referring to the procedure above for retroactively signing the commit once bumped.

**Comment by creviera** [comment]
Added some new doc information around the use of dev-shell to make sure dch is installed (for non-debian users).

File: docs/development/release_management.rst [hunk 6]
```diff
@@ -222,10 +222,24 @@ Release Process
...
+``-x`` flag.
+
+Now you're ready to bump the SecureDrop version on your new branch. There are a bunch of version
+files that'll need to be updated in order to set up the upgrade test for the next release. We do
+this by running the version-updater script and specifying the new version number, which will be the
+next minor version with ``~rc1`` appended. For example, if the release is 1.3.0, then you'll run: 
+``securedrop/bin/dev-shell ../update_version.sh 1.4.0~rc1``  (``dev-shell`` is a script that starts
+a container so that we can ensure ``dch`` is installed). Accept all the default changes from the
+``update_version.sh`` script. You'll only need to add your commit message. Once you're done, sign
+your commit and make a PR to merge these hanges into ``develop``.
```
Comment (eloquence) [inline docs/development/release_management.rst:241]: typo: hanges->changes

**Pull request merged**